### DISCUSSION TITLE
SMB Dashboard

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#4737] [Unsupported Versions]
Running SO 2.3.60

On the SMB Dashboard, on the Security Onion - SMB - Service and Security Onion - SMB - File System, I have 'Error' status. 

SMB Service, has this error: [esaggs] > The field "smb.service.keyword" associated with this object no longer exists in the index pattern. Please use another field.

SMB File System, has the error: [esaggs] > The field "smb.file_system.keyword" associated with this object no longer exists in the index pattern. Please use another field.

See attached example:
![image](https://user-images.githubusercontent.com/519034/124923225-3d4f6a00-dfc8-11eb-8935-5bf513cb5938.png)

**Comment by weslambert** [comment]
Thanks! We'll look into this.

**Reply by Nicalicious** [reply]
Any update on this? I'm seeing this exact error on several of the dashboards within Kibana. RDP/SMB/Intel/IRC/Modbus/MySQL/NTLM/PE/RADIUS/RDP/and so on...

-Fresh install from latest .iso 2.3.61
-ran sudo soup
-replaced certificate with self-signed certificate /etc/pki/managerssl.crt and /etc/pki/managerssl.key

Everything else is default installation for Standalone.

-vcenter/esxi build
-2 NICs /one management / one for promiscuous mode
-32GB RAM
-24 cpu xeon X5670
-SSD storage
